jquantsapi/client.py

```diff
@@ -2,12 +2,14 @@
 from datetime import datetime
 from typing import Dict
 
+from typing import Union
```

Comment:
typing でまとめると良いと思いました！

```suggestion
from typing import Dict, Union

```

Reply:
失礼しました、見落としていました 🙇 対応します。